tidb-cloud/manage-user-access.md

```diff
@@ -119,7 +88,63 @@ To invite a member to an organization, perform the following steps:
 >
 > The verification link in the email will expire in 24 hours. If your user doesn't receive the email, click **Resend**.
 
-## Invite a project member
+### Remove an organization member
+
+If you are the owner of an organization, you can remove organization members. Otherwise, skip this section.
+
+To remove a member from an organization, perform the following steps:
+
+> **Note:**
+>
+> If a member is removed from an organization, the member is removed from the belonged projects either.
+
+1. Click <MDSvgIcon name="icon-top-organization" /> **Organization** in the upper-right corner of the TiDB Cloud console.
+
+2. Click **Organization Settings**. The organization settings page is displayed.
+
+3. Click **User Management**, and then select the **By All Users** tab.
+
+4. Click **Delete** in the user row that you want to delete.
+
+## Manage project access
+
+### Create a project
+
+To create a new project, perform the following steps:
+
+1. Click <MDSvgIcon name="icon-top-organization" /> **Organization** in the upper-right corner of the TiDB Cloud console.
+
+2. Click **Organization Settings**. The **Projects** tab is displayed by default.
+
+3. Click **Create New Project**.
+
+4. Enter your project name.
+
+5. Click **Confirm**.
+
+### View projects
+
+To check which project you belong to, perform the following steps:
+
+1. Click <MDSvgIcon name="icon-top-organization" /> **Organization** in the upper-right corner of the TiDB Cloud console.
+
+2. Click **Organization Settings**. The **Projects** tab is displayed by default.
+
+### Rename a project
+
+To rename a project, perform the following steps:
+
+1. Click <MDSvgIcon name="icon-top-organization" /> **Organization** in the upper-right corner of the TiDB Cloud console.
+
+2. Click **Organization Settings**. The **Projects** tab is displayed by default.
+
+3. In the row of your project to be renamed, click **Rename**.
+
+4. Enter a new project name.
+
+5. Click **Confirm**.
+
+### Invite a project member
```

Comment:
This operation involves 8 steps, which is too long. Better to shorten the steps.